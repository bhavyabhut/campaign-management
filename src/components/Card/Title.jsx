import React from 'react';
import PropTypes from 'prop-types';
import style from './content.module.css';

const Title = ({ campaign_name }) => (
  <div className={style.mrb}>
    <span className={style.customLabel}>Creator Name: </span>
    <span>{campaign_name}</span>
  </div>
);

Title.propTypes = {
  campaign_name: PropTypes.string.isRequired,
};

export default Title;
