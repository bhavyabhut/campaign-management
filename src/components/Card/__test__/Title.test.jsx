import React from 'react';
import ReactDom from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Title from '../Title';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('Renders without crashing', () => {
  const div = document.createElement('div');
  ReactDom.render(<Title></Title>, div);
});

it('Card Title Found match snapshot', () => {
  const tree = renderer.create(<Title campaign_name="Push Notification" />).toJSON();
  expect(tree).toMatchSnapshot();
});
