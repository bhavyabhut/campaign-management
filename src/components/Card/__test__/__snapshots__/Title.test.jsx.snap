// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Title Found match snapshot 1`] = `
<div
  className="mrb"
>
  <span
    className="customLabel"
  >
    Creator Name: 
  </span>
  <span>
    Push Notification
  </span>
</div>
`;
